scripts/bootloader/provision.py

```diff
@@ -29,9 +29,8 @@ def parse_args():
     parser = argparse.ArgumentParser(
         description="Generate provision hex file.",
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument("--s0-addr", type=lambda x: int(x, 0), required=True, help="Set address of s0")
-    parser.add_argument("--s1-addr", type=lambda x: int(x, 0), required=False,
-                        help="Set address of s1. Will default to value of --s0-addr if not set.")
+    parser.add_argument("--s0-addr", type=lambda x: int(x, 0), required=True, help="Address of s0")
```

Comment:
Since we are here, I suggest adding a tiny bit of context.
```suggestion
    parser.add_argument("--s0-addr", type=lambda x: int(x, 0), required=True, help="Address of bootloader image slot s0.")
```

Reply:
It might not be a bootloader. It could be the application